lib/settings.py

```diff
@@ -212,47 +190,50 @@ def prompt(question, choices):
 
 def download_rand_wordlist(verbose=False, multi=1):
     """
-      Download a random wordlist from some wordlists I have laying around
+      Download a random wordlist from some wordlist_links I have laying around
 
       > :param b64link: a base64 encoded wordlist link
     """
-    if multi == 1:
-        b64link = random.choice(WORDLIST_LINKS)
-        filename = "Download-" + random_salt_generator(use_string=True)[0]
-        LOGGER.info("Beginning download..")
-        with open("{}.txt".format(filename), "a+") as wordlist:
-            response = requests.get(base64.b64decode(b64link), stream=True)
-            total_length = response.headers.get('content-length')
-            if verbose:
-                LOGGER.debug("Content length to be downloaded: {}..".format(convert_file_size(int(total_length))))
-                LOGGER.debug("Wordlist link downloading from: '{}'..".format(b64link))
-            if total_length is None:
-                wordlist.write(response.content)
-            else:
-                start = time.time()
+    with open("{}/lib/data_files/wordlist_links".format(os.getcwd())) as wordlist_links:
+        if multi == 1:
+            b64link = random.choice(wordlist_links.readlines())
```

Comment:
No need for that right now. We can however, add a `set()` to it and make sure the link isn't in there before continuing